docs/android/v2/changelog/release-notes.mdx

```diff
@@ -3,6 +3,17 @@ title: Release Notes
 nav: 15
 description: Release Notes for 100ms Android SDK
 ---
+## v2.5.4 - 2022-12-15
+### Added
```

Comment:
these 2 need more detail and links to docs. how would our customers know what "auto simulcast" means?

Reply:
Added